Translator Script/Translation.py

```diff
@@ -0,0 +1,27 @@
+

```

Comment:
Remove the unnecessary blank lines and comments. It's 6 lines code and it will remain that only